### ISSUE TITLE
The specified module could not be found

### ISSUE THREAD

**Issue by alex-gillott** [killpowa/Twickt-Launcher#1328]
I just reinstalled Windows on my laptop, and tried to install GDLauncher. I've redownloaded the installer, deleting all folders I could find related to GDLauncher between installs (In gdlauncher in Roaming and Local), yet whenever I try to open the launcher I get this error message. The instance then stays open in  the background and has to be ended via task manager or similar.

![image](https://user-images.githubusercontent.com/20886909/167255999-daa1422a-3d85-4fbf-bcab-ee40dec20c09.png)
I thought it might be related to #823, but the main error in that issue was that access was being denied by something on their PC, where this error is reporting that some module couldn't be found.

It might help to know that I copied over my User folder (was called banan) from my previous install of Windows by exporting to an external hard drive, as I was replacing the hard drive in this laptop but wanted to keep my files. I don't know if this broke anything related to GDLauncher, but I'm assuming it did considering this error is happening

**Comment by YoYoK3nny** [comment]
Sounds like a permissions issue, commonly caused by improper user migration.

There are potentially a lot more things broken that you just haven't encountered yet.

I find it works best to copy the files to a drive that doesn't store permissions (ie: not formatted to NTFS) then copy to the new install.
This way the permissions of the new User folder should be applied to the copied files, instead of keeping the invalid permissions from the old User folder.
It does work best if certain files that don't need to be copied are removed before copying to the new location.

tl;dr: Resetting the permissions on your `AppData\Local\Temp` folder may help.

P.S: Might be a good idea to go to your `C:\Users` folder, right-click on `purpl`, click `Properties`, click `Security`, click `Advanced`, check the `Replace all child object permission entries with inheritable permission entries from this object` and click `apply`.
Some `Access is denied` prompts are normal in this process, just hold the `C` key until it is done.
Don't mess with anything else in there, close when done.
If problems persist, try again in `Safe Mode`

**Comment by Eskaan** [comment]
This is a common issue if the install/update is interrupted. Reinstall the App from the releases, then it should work
